### ISSUE TITLE
Review the button's wording rule in the checkout flow

### ISSUE THREAD

**Issue by ElodieOS** [PrestaShop/hummingbird#519]
Hi everyone! 

In the checkout flow, the button's wording rule varies depending on the button.
We have checked with @l-delin and we think it will be better to harmonize them with "Continue to [step name]" and an icon. It's better for guiding the user (except for the form validation buttons, which are there to validate the form and not to move on to the next step).

![image](https://github.com/PrestaShop/hummingbird/assets/113457393/85651f3c-50d3-4cb7-aaac-dddcd7505148)

![image](https://github.com/PrestaShop/hummingbird/assets/113457393/572dbecb-7b30-493c-b2eb-5c506f48bafc)

**Comment by Hlavtox** [comment]
Good point, but it MAY require some core changes I think, because the address form in the second step is unified with the address form in customer account, having a "Save" button.

**Comment by ElodieOS** [comment]
Yes, but if we keep the save button in the address form, can it work?
